lib/css/exports/_stacks-constants-colors.less

```diff
@@ -198,7 +198,7 @@
 
 .component-colors {
     --theme-body-font-color: var(--black-800);
-
+    --theme-background-color: var(--white);
```

Comment:
lgtm. This should "Just Work™" in Core, but I'll be sure to test for regressions there.